src/moveit_opw_kinematics_plugin.cpp

```diff
@@ -497,34 +497,48 @@ bool MoveItOPWKinematicsPlugin::setOPWParameters()
 
   ros::NodeHandle nh;
 
-  std::map<std::string, double> geometric_parameters, dummy;
-  if (!lookupParam("opw_kinematics_geometric_parameters", geometric_parameters, dummy))
+  std::map<std::string, double> geometric_parameters;
+  if (!lookupParam("opw_kinematics_geometric_parameters", geometric_parameters, {}))
   {
     ROS_ERROR_STREAM("Failed to load geometric parameters for ik solver.");
     return false;
   }
 
-  std::vector<double> joint_offsets, dummy2;
-  if (!lookupParam("opw_kinematics_joint_offsets", joint_offsets, dummy2))
+  std::vector<double> joint_offsets;
+  if (!lookupParam("opw_kinematics_joint_offsets", joint_offsets, {}))
   {
     ROS_ERROR_STREAM("Failed to load joint offsets for ik solver.");
     return false;
   }
 
-  std::vector<int> joint_sign_corrections, dummy3;
-  if (!lookupParam("opw_kinematics_joint_sign_corrections", joint_sign_corrections, dummy3))
+  std::vector<int> joint_sign_corrections;
+  if (!lookupParam("opw_kinematics_joint_sign_corrections", joint_sign_corrections, {}))
   {
     ROS_ERROR_STREAM("Failed to load joint sign corrections for ik solver.");
     return false;
   }
 
   opw_parameters_.a1 = geometric_parameters["a1"];
   opw_parameters_.a2 = geometric_parameters["a2"];
-  opw_parameters_.b = geometric_parameters["b"];
+  opw_parameters_.b  = geometric_parameters["b"];
```

Comment:
I noticed this is going to show up in future commits due to clang format. But I want this change because it IS really nicer if everything is aligned :)

Reply:
Hm. If we want to conform to Clang format, then we shouldn't introduce formatting errors on purpose.

I like alignment as much as anyone (I guess), but let's not fight something which is intended to make our live easier.